sharing-data.html

```diff
@@ -389,6 +390,112 @@
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="share"></a>Sharing data with users you are not in a group with</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<ol>
+
+<li><p class="first">In OMERO.web, you can share data with selected other users in different groups by adding it to your <span class="bold">Public</span> list (the other users must have accounts on the same OMERO server).</p>
+
+<p>Select the image(s) you wish to share.</p>
+
+<p>Click on the <span class="bold">Globe</span> icon in the toolbar.</p>
+
+<p>or right-click on the selected images in the data tree and select <span class="bold">Create share</span>.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/webclientShare.jpg" class="align-center" src="images/webclientShare.jpg" style="width: 80%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li><p class="first">Enter a message in the <span class="bold">Message</span> field.</p>
+
+<p>This message will be sent automatically via email to all share members.</p>
+
+<p>An expiry date can be set.</p>
+
+<p>Start typing the name of a user to add to the share in the <span class="bold">Members</span> field.</p>
+
+<p>Select users to add to the share from the filtered list of matching users.</p>
+
+<p>Click <span class="bold">Accept</span> to complete the creation of the share.</p>
+
+<p>All the members of the share who have an email address set on OMERO.server will receive email notification of the share.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/webclientCreate.jpg" class="align-center" src="images/webclientCreate.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">The share will appear in the <span class="bold">Public</span> tab.</p>
+
+<p>The details of the share can be viewed in the right hand pane.</p>
+
+<p>Click on the green <span class="bold">Edit</span> icon edit to change the message, expiry date, or add/remove users with access to the share.</p>
```

Comment:
'...Edit icon edit..' reads somewhat strangely, you don't need the second 'edit'